packages/eslint-config-widen-emotion/test/no-conditional-css-prop.test.js

```diff
@@ -0,0 +1,57 @@
+import { RuleTester } from 'eslint'
+import rule from '../src/rules/no-conditional-css-prop'
+import sharedGlobals from '../src/sharedGlobals'
+
+const ruleTester = new RuleTester({
+  languageOptions: {
+    globals: sharedGlobals,
+    parserOptions: {
+      ecmaFeatures: {
+        jsx: true,
+      },
+    },
+  },
+})
+
+ruleTester.run('no-conditional-css-prop', rule, {
+  invalid: [
+    {
+      code: '<div css={condition ? { color: "red" } : { color: "blue" }} />',
+      errors: [
+        {
+          message:
+            'Avoid using conditionals within the css prop, move them to style.',
+        },
+      ],
+    },
+    {
+      code: '<div css={condition && { color: "red" }} />',
+      errors: [
+        {
+          message:
+            'Avoid using conditionals within the css prop, move them to style.',
+        },
+      ],
+    },
+    {
+      code: '<div css={condition || { color: "blue" }} />',
+      errors: [
+        {
+          message:
+            'Avoid using conditionals within the css prop, move them to style.',
+        },
+      ],
+    },
+  ],
+  valid: [
+    {
+      code: '<div css={{ color: "red" }} />',
+    },
+    {
+      code: '<div css={someVariable} />',
```

Comment:
Can you also add a check for an array of styles? Sometimes we add a base and then an extra style.

Reply:
Do we need some valid style usages, so that devs understand what we mean by move conditions to style?

Reply:
[722368b](https://github.com/Widen/eslint-config/pull/50/commits/722368bbc85bcee76e5eea3c61f60cc3f76dbe72)